### ISSUE TITLE
Result Summary Table GHG -> add the substages

### ISSUE THREAD

**Issue by sebakoe** [icra/ecam#480]
Not sure if there was already a discussion around this, but whereas the results summary table for energy consumption lists the substage, the results summary table for GHG emissions does not list the subtages. This would be pretty helpful for the user to figure out through that summary table in which substage which emission source occurs.

**Comment by holalluis** [comment]
Hi, this is because in the summary we list the sum of the sources for each substage. However, you can have this information in the menu were you select the substage:
![image](https://user-images.githubusercontent.com/1082644/132566574-e7c59429-19fd-488b-abb3-a860efda8955.png)

**Comment by holalluis** [comment]
I agree that this can be improved having a list of the substages (for example) here:
![image](https://user-images.githubusercontent.com/1082644/132566874-bd675891-38dd-44f8-bc27-dd183ec146da.png)

**Comment by holalluis** [comment]
Remember that in the summary, next to the Table, you have the button "Charts GHG" were you can see a bar chart of the GHG splitted by substage
![image](https://user-images.githubusercontent.com/1082644/132567120-5ad5c02f-863a-4b19-bc5a-f36998c367dc.png)
